_data/simple-icons.json

```diff
@@ -2437,6 +2437,11 @@
                 "url": "https://logorequest.broadcom.com/Home.aspx"
             }
         },
+        {
+            "title": "Bruno",
```

Comment:
Wow Bruno might be the least consistent project/brand I've ever seen in terms of capitalization of their name - their app, their website title, and their logo all use "bruno" but then their documentation, branding, etc all use "Bruno"...

Reply:
Let's play it safe and go with the capitalised name then.